lib/dnsimple/client/registrar.rb

```diff
@@ -41,6 +41,27 @@ def register(account_id, domain_name, attributes = {}, options = {})
         Dnsimple::Response.new(response, Struct::Domain.new(response["data"]))
       end
 
+      # Renews a domain.
+      #
+      # @see https://developer.dnsimple.com/v2/registrar/#renew
+      #
+      # @example Renew example.com for 3 years:
+      #   client.registrar.renew(1010, "example.com", period: 3)
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [#to_s] domain_name The domain name to register.
+      # @param  [Hash] attributes
```

Comment:
To be honest I was just C&Ping from an existing commend and I didn't pay much attention to the format or the explanation for `attributes` or `options`.

Are you ok with me updating this as part of another PR for every registrar method? (I will open one no matter what since I want to append the `_domain` to every endpoint as discussed yesterday... Maybe you only meant the `renew` endpoint?)...